database/sqlite/sqlite_metadata.c

```diff
@@ -354,7 +354,8 @@ static int store_host_metadata(RRDHOST *host)
     if (unlikely(rc != SQLITE_OK))
         goto bind_fail;
 
-    rc = bind_text_null(res, ++param, rrdhost_tags(host), 1);
+    // TODO: remove/migrate
```

Comment:
@stelfrag Okay for you to remove this whenever the need arises to perform the next migration? I'm not familiar with the code/component that performs the migrations and considering the simplicity of the PR, I didn't want to add more complexity.